tools/android_lint/bin/main.dart

```diff
@@ -0,0 +1,120 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
2019?

Reply:
I'm worried the license checker won't be smart enough to figure out this doesn't impact licensing :(

Reply:
FYI @gspencergoog @dnfield my understanding is that 2013 should be the year used on all licenses within the embedding.

Reply:
Nevermind, this file lives somewhere else.